newIDE/app/src/Utils/GDevelopServices/Usage.js

```diff
@@ -354,18 +388,26 @@ export const getRedirectToSubscriptionPortalUrl = (
     });
 };
 
-export const getRedirectToCheckoutUrl = (
+export const getRedirectToCheckoutUrl = ({
+  planId,
+  userId,
+  userEmail,
+  quantity,
+}: {
   planId: string,
   userId: string,
-  userEmail: string
-): string => {
-  return `${
-    GDevelopUsageApi.baseUrl
-  }/subscription-v2/action/redirect-to-checkout?planId=${encodeURIComponent(
-    planId
-  )}&userId=${encodeURIComponent(userId)}&customerEmail=${encodeURIComponent(
-    userEmail
-  )}`;
+  userEmail: string,
+  quantity?: number,
+}): string => {
+  const url = new URL(
+    `${GDevelopUsageApi.baseUrl}/subscription-v2/action/redirect-to-checkout`
+  );
+  url.searchParams.set('planId', planId);
+  url.searchParams.set('userId', userId);
+  url.searchParams.set('customerEmail', userEmail);
```

Comment:
just to double check, are we sure we don't need to use `encodeURIComponent()` with this function?